### PR TITLE
Adds fib agents as ghost roles for the cciaa

### PR DESCRIPTION
**Pull request by Alberyk**
The cciaa head asked for this.

### CODE REVIEW

File: code/datums/outfits/outfit_admin.dm [hunk 1]
```diff
@@ -273,6 +273,47 @@
...
+	accessory_contents = list(/obj/item/gun/projectile/sec/lethal = 1)
+
+	backpack_contents = list(
+		/obj/item/storage/box/engineer = 1,
+		/obj/item/device/flash = 1,
+		/obj/item/handcuffs = 1
+	)
+
+/datum/outfit/admin/nt/fib/guard
+	name = "FIB Escord"
```
Comment (Geevies) [inline code/datums/outfits/outfit_admin.dm]:
```suggestion
	name = "FIB Escort"
```

File: code/modules/clothing/suits/armor.dm [hunk 2]
```diff
@@ -85,6 +85,12 @@
 	icon_state = "commissar_warden"
 	item_state = "commissar_warden"
 
+/obj/item/clothing/suit/armor/vest/fib
+	name = "fib armored vest"
```
Comment (Geevies) [inline code/modules/clothing/suits/armor.dm]:
```suggestion
	name = "\improper FIB armored vest"
```

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -625,6 +625,12 @@
 	icon_open = "sec_dep_jacket_open"
 	icon_closed = "sec_dep_jacket"
 
+/obj/item/clothing/suit/storage/fib
+	name = "fib agent jacket"
```
Comment (Geevies) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	name = "\improper FIB agent jacket"
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 4]
```diff
@@ -99,6 +99,12 @@
 	worn_state = "centcom"
 	displays_id = 0
 
+/obj/item/clothing/under/rank/fib
+	name = "fib agent uniform"
```
Comment (Geevies) [inline code/modules/clothing/under/miscellaneous.dm]:
```suggestion
	name = "\improper FIB agent uniform"
```

File: code/datums/outfits/outfit_admin.dm [hunk 5]
```diff
@@ -273,6 +273,47 @@
...
+/datum/outfit/admin/nt/fib
+	name = "FIB Agent"
+
+	uniform = /obj/item/clothing/under/rank/fib
+	suit = /obj/item/clothing/suit/storage/fib
+	gloves = /obj/item/clothing/gloves/black
+
+	l_pocket = /obj/item/reagent_containers/spray/pepper
+	r_pocket = /obj/item/device/taperecorder/cciaa
+	l_hand = /obj/item/storage/lockbox/cciaa
```
Comment (fernerr) [inline code/datums/outfits/outfit_admin.dm]: I think they should either get no lockbox or one rebranded to FIB
```suggestion
	l_hand = /obj/item/storage/lockbox/cciaa
```

**Pull request closed without merging**